Specs/DateTime/English/DateTimeModel.json

```diff
@@ -20627,5 +20627,80 @@
         }
       }
     ]
+  },
+  {
+    "Input": "etfs with negative flows of at least $1k last month",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "last month",
+        "Start": 41,
+        "End": 50,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2016-10",
+              "type": "daterange",
+              "start": "2016-10-01",
+              "end": "2016-11-01"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "We will be only 3 next week.",
```

Comment:
Can you add some cases with the plurals too, if they don't exists? Like "We'll be gone for only the 3 next weeks".
Please add the corrected support specs in a couple other languages too.

Reply:
added

Reply:
also in French, Portuguese and Spanish